platforms/config_settings/BUILD

```diff
@@ -0,0 +1,57 @@
+load("//platforms/config_settings:recycle_groups.bzl", "BUILDBUDDY_RECYCLE_GROUPS")
+
+package(default_visibility = ["//visibility:public"])
+
+config_setting(
+    name = "linux_x86_64",
+    constraint_values = [
+        "@platforms//os:linux",
+        "@platforms//cpu:x86_64",
+    ],
+)
+
+config_setting(
+    name = "macos_x86_64",
+    constraint_values = [
+        "@platforms//os:macos",
+        "@platforms//cpu:x86_64",
+    ],
+)
+
+config_setting(
+    name = "macos_arm64",
+    constraint_values = [
+        "@platforms//os:macos",
+        "@platforms//cpu:arm64",
+    ],
+)
+
+constraint_setting(name = "sandbox")
```

Comment:
Initially I named this constraint_setting to be `isolation` because I plan to have this represent the isolation technologies we have: firecracker, podman, docker, bare, linux-sandbox etc…

However I thought `isolation` might be too low level of a word to expose to BuildBuddy user… so I was thinking to changing this to `sandbox` or `sandbox-technology` instead.

But I do like your suggestion 🤔 Perhaps isolation should only be the implementation detail underneath and not the higher level interface.

Reply:
> Perhaps isolation should only be the implementation detail underneath and not the higher level interface.

Yeah agreed, that's a good way to put it